cpp/src/prims/extract_transform_v_frontier_outgoing_e.cuh

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2020-2022, NVIDIA CORPORATION.
```

Comment:
Is this new (should be just 2022) or moved from somewhere?

Reply:
Yes, this is actually new. I will fix the copyright year.